### ISSUE TITLE
Simpler, more professional console messages

### ISSUE THREAD

**Issue by gaelanlloyd** [lando/lando#3293]
Awesome project, Lando team!

The console messages are fun and cute, but after a while the ASCII art takes up a lot of space and are a distraction.

In line with #1726, is there a way to make the messages simpler and more professional?

For instance, when shutting down:

```
% lando poweroff

NO!! SHUT IT ALL DOWN!!!
   ___                     _                  ______
  / _ \___ _    _____ ____(_)__  ___ _  ___  / _/ _/
 / ___/ _ \ |/|/ / -_) __/ / _ \/ _ `/ / _ \/ _/ _/ _ _
/_/   \___/__,__/\__/_/ /_/_//_/\_, /  \___/_//_/(_|_|_)
                               /___/

Bye bye landoproxyhyperion5000gandalfedition_proxy_1 ...
Bye bye mylandoapp_database_1 ...
Bye bye mylandoapp_appserver_1 ...
Lando containers have been spun down.
```

Could be simplified and cleaned up to be:

```
% lando poweroff

Shutting down Lando containers:
  - landoproxyhyperion5000gandalfedition_proxy_1 ...
  - mylandoapp_database_1 ...
  - mylandoapp_appserver_1 ...
Done
```

I'm thinking there could be an advanced mode that quietly closes the containers, listing them only if there's an error.

```
% lando poweroff
Shutting down Lando containers... Done
```

There could even be a PRO mode that is silent unless there are errors.  The idea here is that we just trust Lando to do its job and don't need to know all of the details unless we specify that we want the added verbosity.

```
% lando poweroff
%
```

I'm not sure if `logLevelConsole` affects this, I can't really find any documentation on what it does or what the various settings are other than setting it to `silly` [in this example](https://docs.devwithlando.io/config/global.html#set-a-bunch-of-custom-stuff-using-a-yaml-file).

It would be nice if this was a setting in the global Lando config so it would apply to all Lando commands in a user's environment.

**Comment by reynoldsalec** [comment]
There have been a few varying requests over time to give options to "tone down" the messages. Lando is looking at switching to a new CLI tooling (https://oclif.io/), which may make it easier to accomplish this, although it's probably a low priority compared to many of the other 4.x roadmap items.

I think this may be a duplicate of a previous issue but I couldn't find one, so going to keep this open for now with the 4.x reminder label.

**Comment by trackleft** [comment]
One could argue that these giant messages are actually useful when scrolling through your history and visually break up the commands you have run.

**Comment by reynoldsalec** [comment]
Yeah, I personally like the BIG messages, both to break up output and also for their aesthetics (gives me that cyberpunk vibe that I'm looking for sometimes).

If someone ever gets the chance to tackle this, I think having a setting (`corporatemode`? `buttondown`?) that would manipulate output to satisfy the bossman would be good.

**Comment by pirog** [comment]
@stebogit i think this ticket is more about the style and theme of the output and less about the verbosity of them

**Comment by stebogit** [comment]
yeah maybe you're right; what I had in mind was an additional flag, something like
```sh
$ lando poweroff --quiet
Shutting down myapp... Done
```

```sh
$ lando poweroff --silent
$
```
and 
```sh
$ lando start --quiet
Starting myapp... Started!
```

```sh
$ lando start --silent
$
```

Maybe a different issue then?

**Comment by pirog** [comment]
@stebogit yeah i would open another ticket for that in https://github.com/lando/core-next and im also going to move this one over there because both of these are def lando 4 things.

i also would say that im open to `--silent`, assuming that its purpose is to suppress output completely but the case would have to be pretty compelling given that a user can do this on their own with redirects, pipes, etc. 

im less open to `--quiet` assuming it means "less output" because its not obvious to me WHAT should be suppressed and it _feels_ pretty gross to me to have both `--silent` and `--quiet`. 

in my ideal world and in lando 4 we just have better output so `--quiet` isnt even something we feel we need and `--silent` maybe is covered by one of the "languages" we plan on supporting that you could switch to.
